app/models/manageiq/providers/ansible_tower/automation_manager/credential.rb

```diff
@@ -1,3 +1,7 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::Credential < ManageIQ::Providers::ExternalAutomationManager::Authentication
+  alias_attribute :manager_id, :resource_id
+  alias_attribute :manager, :resource
```

Comment:
`Authentication` is joined to `manager` through the polymorphic `resource_id/resource_type` and that makes `credential` 's [this call](https://github.com/jameswnl/manageiq/blob/d447477819338ebcb8e755f08b6dadad4992af39/app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb#L10-L10) different from that of `css` and `cs` (coming in next PR).

I'd love to hear better suggestion on this.